<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check Deflection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../civiltools.qrc">
    <normaloff>:/civiltools/images/deflection.svg</normaloff>:/civiltools/images/deflection.svg</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Dead</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QComboBox" name="dead_combobox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="sdead_combobox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="partition_combobox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Dead</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Supper-Dead</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>Partition</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Live</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1">
         <widget class="QComboBox" name="live5_combobox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="lred_combobox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="live_combobox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Live</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>L-RED</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>Live_0.5</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Roof</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <widget class="QComboBox" name="lroof_combobox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="snow_combobox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>LROOF</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>SNOW</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="6" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Live Load Percentage</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Cover</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QDoubleSpinBox" name="live_percentage_spinbox">
         <property name="suffix">
          <string/>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
         <property name="value">
          <double>0.250000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="torsion_rebar_checkbox">
         <property name="text">
          <string>Add Torsion Rebars</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="is_console">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Is Console</string>
         </property>
         <property name="icon">
          <iconset resource="../../civiltools.qrc">
           <normaloff>:/civiltools/images/console.svg</normaloff>:/civiltools/images/console.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>Add Rebar</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="cover_spinbox">
         <property name="suffix">
          <string> cm</string>
         </property>
         <property name="value">
          <double>6.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QDoubleSpinBox" name="additional_rebars">
         <property name="suffix">
          <string> cm2</string>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="maximum">
          <double>500.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QDoubleSpinBox" name="minus_length">
         <property name="suffix">
          <string> cm</string>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="maximum">
          <double>500.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>5.000000000000000</double>
         </property>
         <property name="value">
          <double>30.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>Minus Length</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="beam_dimension_groupbox">
       <property name="toolTip">
        <string>For Non prismitic sections</string>
       </property>
       <property name="title">
        <string>Beam Dimensions</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Width</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="beam_width_spinbox">
          <property name="suffix">
           <string> Cm</string>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="maximum">
           <double>500.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>5.000000000000000</double>
          </property>
          <property name="value">
           <double>50.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="beam_height_spinbox">
          <property name="suffix">
           <string> Cm</string>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="maximum">
           <double>500.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>5.000000000000000</double>
          </property>
          <property name="value">
           <double>50.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <widget class="QPushButton" name="check_button">
         <property name="text">
          <string>Check</string>
         </property>
         <property name="icon">
          <iconset resource="../../civiltools.qrc">
           <normaloff>:/civiltools/images/tick.svg</normaloff>:/civiltools/images/tick.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="cancel_button">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
         <property name="icon">
          <iconset resource="../../civiltools.qrc">
           <normaloff>:/civiltools/images/cancel.svg</normaloff>:/civiltools/images/cancel.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="open_main_file_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Open Main File</string>
         </property>
         <property name="icon">
          <iconset resource="../../civiltools.qrc">
           <normaloff>:/civiltools/images/open.svg</normaloff>:/civiltools/images/open.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="help">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Help</string>
         </property>
         <property name="icon">
          <iconset resource="../../civiltools.qrc">
           <normaloff>:/civiltools/images/help.png</normaloff>:/civiltools/images/help.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTextEdit" name="check_results">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QTextEdit" name="results">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>is_console</tabstop>
  <tabstop>torsion_rebar_checkbox</tabstop>
  <tabstop>cover_spinbox</tabstop>
  <tabstop>live_percentage_spinbox</tabstop>
  <tabstop>check_button</tabstop>
  <tabstop>dead_combobox</tabstop>
  <tabstop>sdead_combobox</tabstop>
  <tabstop>partition_combobox</tabstop>
  <tabstop>live_combobox</tabstop>
  <tabstop>lred_combobox</tabstop>
  <tabstop>live5_combobox</tabstop>
  <tabstop>lroof_combobox</tabstop>
  <tabstop>snow_combobox</tabstop>
  <tabstop>cancel_button</tabstop>
  <tabstop>help</tabstop>
  <tabstop>open_main_file_button</tabstop>
 </tabstops>
 <resources>
  <include location="../../civiltools.qrc"/>
 </resources>
 <connections/>
</ui>
